OrbitGl/TimeGraph.cpp

```diff
@@ -458,6 +462,8 @@ inline std::string GetExtraInfo(const Timer& a_Timer) {
 
 //-----------------------------------------------------------------------------
 void TimeGraph::UpdatePrimitives(bool a_Picking) {
+  CHECK(string_manager_);
```

Comment:
```suggestion
  CHECK(string_manager_ != nullptr);
```
Let's make it explicit.

Reply:
shared_ptr has a boolean operator specifically for this case, and it's more idiomatic to use that in checks.

Reply:
As you wish, but I don't agree. Many types have boolean operators, it doesn't mean it is good to use them instead of making your intentions explicit (of course this is a very basic case). It's the same for bare pointers, one can use them as booleans or compare to nullptr, and I just dislike the first option. Even the message that one would get in case of a failure would be better ("Check failed: string_manager_" vs "Check failed: string_manager_ != nullptr").

Reply:
I'd prefer to keep as-is.